cmd/sources_controller/main.go

```diff
@@ -17,182 +17,17 @@ limitations under the License.
 package main
 
 import (
-	"flag"
-	"log"
-
-	"github.com/knative/eventing/pkg/metrics"
-
 	// Uncomment the following line to load the gcp plugin (only required to authenticate against GKE clusters).
 	// _ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 
-	informers "github.com/knative/eventing/pkg/client/informers/externalversions"
-	"github.com/knative/eventing/pkg/logconfig"
-	"github.com/knative/eventing/pkg/reconciler"
-	"github.com/knative/eventing/pkg/reconciler/apiserversource"
-	"github.com/knative/eventing/pkg/reconciler/containersource"
-	"github.com/knative/eventing/pkg/reconciler/cronjobsource"
-	"github.com/knative/pkg/configmap"
-	kncontroller "github.com/knative/pkg/controller"
-	"github.com/knative/pkg/logging"
-	"github.com/knative/pkg/logging/logkey"
-	pkgmetrics "github.com/knative/pkg/metrics"
-	"github.com/knative/pkg/signals"
-	"go.uber.org/zap"
-	kubeinformers "k8s.io/client-go/informers"
-	"k8s.io/client-go/rest"
-	"k8s.io/client-go/tools/clientcmd"
-)
+	_ "github.com/knative/eventing/pkg/reconciler/apiserversource"
```

Comment:
This is super magical, I'm not sure how I feel about it.

Reply:
it will be ok.